pkg/ebpf/tracer.go

```diff
@@ -206,11 +206,7 @@ func NewTracer(config *Config) (*Tracer, error) {
 	portMapping := network.NewPortMapping(config.ProcRoot, config.CollectTCPConns, config.CollectIPv6Conns)
 	udpPortMapping := network.NewPortMapping(config.ProcRoot, config.CollectTCPConns, config.CollectIPv6Conns)
 	if err := portMapping.ReadInitialState(); err != nil {
-		return nil, fmt.Errorf("failed to read initial TCP pid->port mapping: %s", err)
-	}
-
-	if err := udpPortMapping.ReadInitialUDPState(); err != nil {
-		return nil, fmt.Errorf("failed to read initial UDP pid->port mapping: %s", err)
+		return nil, fmt.Errorf("failed to read initial pid->port mapping: %s", err)
```

Comment:
Is portMapping now meant to hold listening ports for UDP and TCP ?  If so we should remove the `udpPortMapping` field altogether.

I don't think storing udp and tcp portmappings together is right though, there can be listeners on the same port with different transports.

Reply:
No I don't think this is correct. Let me fix it.